src/controllers/userController.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { getMaxId } = require('../utils/helpers.js');
+const usersServices = require('../services/usersServices.js');
+
+const getAllUsers = (req, res) => {
+  const users = usersServices.getAll();
+
+  res.send(users);
+};
+
+const createUser = (req, res) => {
+  const { name } = req.body;
+  const id = getMaxId(usersServices.getAll());
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  const newUser = {
+    id,
```

Comment:
do it in the service
```suggestion
  if (!name) {
    return res.sendStatus(400);
  }

  const newUser = usersServices.create(name);
```